### ISSUE TITLE
Add "Change language" to other than androids default

### ISSUE THREAD

**Issue by Siedlerchr** [ItsCalebJones/SpaceLaunchNow-Android#123]
Hi,

my android OS language is German, therefore the app is partly in German, but I would like to use the app in English. It would be nice to add a language switcher to switch between supported languages.

Regards

**Comment by ItsCalebJones** [comment]
That's an excellent feature request, not sure how to do it. Is there problems with the translation?

**Comment by Siedlerchr** [comment]
No, the translation is okay, but there might be other users who have a default language for example english on their phone and still want to use another variant in the translation.
PS: I found this article which sums up different options and pitfalls: (I'm a Java Dev myself, just the last time I touched android programming was  a very long time ago) https://proandroiddev.com/change-language-programmatically-at-runtime-on-android-5e6bc15c758
